react/IntentModal/IntentModal.md

```diff
@@ -0,0 +1,23 @@
+```
```

Comment:
Can you add a text explaining what's a usecase for this ?

Reply:
Sure!